cgmes/cgmes-model/src/main/java/com/powsybl/cgmes/model/CgmesNamespace.java

```diff
@@ -45,6 +45,7 @@ private CgmesNamespace() {
     private static final String CIM_16_SV_PROFILE = "http://entsoe.eu/CIM/StateVariables/4/1";
     private static final String CIM_16_SSH_PROFILE = "http://entsoe.eu/CIM/SteadyStateHypothesis/1/1";
 
+    private static final String CGMES_EQ_3_OR_GREATER_PREFIX = "http://iec.ch/TC57/ns/CIM/CoreEquipment-EU/";
```

Comment:
Is this used? Does it exist?

Reply:
yes, to check if an EQ file is version 3 or later (when checking if all loaded EQs are CGMES 3), `CgmesModelTripleStore::allEqCgmes3OrGreater`